bin/run.js

```diff
@@ -827,3 +829,10 @@ try {
     throw er
   }
 }
+
+try {
+  process.on('unhandledRejection', onError)
```

Comment:
This is new, should `process.off` be added after the call to jack?

Otherwise this looks fine and I've verified it acts as I expect when `.taprc` is invalid.

Reply:
Turns out that removing the handler right after teh call to jack removes it before it can be relevant, because async function.

Updated branch with another approach that seems to work.

Reply:
Retested with the updated branch, still resolves the issue for me.